common/src/main/java/com/lightstep/tracer/shared/CollectorClient.java

```diff
@@ -31,12 +32,13 @@ public CollectorClient(AbstractTracer tracer, String host, int port) {
    */
   public CollectorClient(AbstractTracer tracer, ManagedChannelBuilder<?> channelBuilder) {
     this.tracer = tracer;
+    this.channelBuilder = channelBuilder;
```

Comment:
I have a slight preference for de-deduplicating these two lines that initialize channel and blockingStub, but I won't insist.